src/azure-cli/azure/cli/command_modules/compute_recommender/aaz/latest/compute_recommender/_spot_placement_recommender.py

```diff
@@ -56,21 +59,28 @@ def _build_arguments_schema(cls, *args, **kwargs):
             options=["--availability-zones"],
             arg_group="SpotPlacementScoresInput",
             help="Defines if the scope is zonal or regional.",
+            is_preview=True,
```

Comment:
Additionally, labeling GA commands as `is_preview` is a recommended  behavior. Is this because the previous PR forgot to mark it as preview?

Reply:
Right. the cli is in public preview stage. So, marking the parameters to public preview too.